apollo-router/src/services/subgraph_service.rs

```diff
@@ -1938,9 +1958,7 @@ mod tests {
             }
         }
 
-        let make_svc = make_service_fn(|_conn| async { Ok::<_, Infallible>(service_fn(handle)) });
-        let server = Server::from_tcp(listener).unwrap().serve(make_svc);
-        server.await.unwrap();
+        serve(listener, handle).await.unwrap();
```

Comment:
this is technically not part of this changeset, but should this be `handler`? `listener` + `handler` seem to be more aligned than `listener` + `handle`

Reply:
i think there is no upside or downside to either `handle` or `handler`.
the listener is an object that is already listening. only a noun works.
`handle` is a function that will be called to handle a request, so a verb works. it's also a thing that handles a request so a noun works too. `handler` might be more aligned but there is no particular reason for these to be aligned.